react/Tile/styles.styl

```diff
@@ -0,0 +1,45 @@
+$tileSize = 8.75rem
+$tileMobileHeight = 3.75rem
+
+.Tile
+  box-sizing border-box
+  position relative
+  display flex
+  flex-direction column
+  flex $tileSize 0 0
+  height $tileSize
+  align-items center
+  background var(--white)
+  border-radius 4px
```

Comment:
rem?

Reply:
Not sure that this is relevant for border-radiuses.